core/src/main/java/org/openedx/core/ui/IAPUI.kt

```diff
@@ -183,7 +183,7 @@ fun NoSkuErrorDialog(
                 onClick = onConfirm
             )
         },
-        onDismissRequest = onConfirm
+        onDismissRequest = {}
```

Comment:
FYI: We could have also set DialogProperties. 😉

Reply:
right, but `onDismissRequest` is mandatory for the definition of the AlertDialog that we are using.

Reply:
for the safe side I'm adding the properties.